### PR TITLE
Replaces alot of list input with tgui_input_list and alert() with Tgui_alert().

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
Replaces alot of input popups with tgui new input UI's, they are way faster and open basically instantly after the first time.

## Why It's Good For The Game
Really good fast UI

## Changelog
:cl:
code: Changes alot of byond input popups with TGUI Input UI's.
/:cl:

### CODE REVIEW

File: code/datums/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -632,7 +632,7 @@ Sensors indicate [numXenosShip || "no"] unknown lifeform signature[numXenosShip
 	if(instant_join)
 		return pick(available_xenos) //Just picks something at random.
 
-	var/mob/living/carbon/xenomorph/new_xeno = input("Available Xenomorphs") as null|anything in available_xenos
+	var/mob/living/carbon/xenomorph/new_xeno = tgui_input_list(usr, title = "Available Xenomorphs", available_xenos)
```
Comment (spookydonut) [inline code/datums/gamemodes/game_mode.dm:635]:
```suggestion
	var/mob/living/carbon/xenomorph/new_xeno = tgui_input_list(usr, null, "Available Xenomorphs", available_xenos)
```

**Comment by SplinterGP** [comment]
Thanks spooky

File: code/modules/client/preferences.dm [hunk 2]
```diff
@@ -863,7 +863,9 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 				valid_facialhairstyles[facialhairstyle] = GLOB.facial_hair_styles_list[facialhairstyle]
 
-			var/new_f_style = input(user, "Choose your character's facial-hair style:", "Facial Hair Style")  as null|anything in (valid_facialhairstyles + "Shaved")
+			var/list/f_hairstyles = valid_facialhairstyles
+			f_hairstyles += "Shaved"
+			var/new_f_style = tgui_input_list(user, "Choose your character's facial-hair style:", "Facial Hair Style", f_hairstyles)
```
Comment (TiviPlus) [inline code/modules/client/preferences.dm:868]: Why?

**Review by TiviPlus — APPROVED** [review]

File: code/modules/admin/fun_verbs.dm [hunk 3]
```diff
@@ -658,11 +662,12 @@
 	if(!check_rights(R_FUN))
 		return
 
-	var/sec_level = input(usr, "It's currently code [GLOB.marine_main_ship.get_security_level()]. Choose the new security level.", "Set Security Level") as null|anything in (list("green", "blue", "red", "delta") - GLOB.marine_main_ship.get_security_level())
+	var/available_levels = list("green", "blue", "red", "delta") - GLOB.marine_main_ship.get_security_level()
```
[the remark this answers is not in the document]
Comment (SplinterGP) [reply code/modules/admin/fun_verbs.dm:665]: Wanted to ask ya, can you inline the list operation just fine?

**Review by TiviPlus — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**